CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 <a name="unreleased"></a>
 ## [Unreleased]
 
+<a name="v0.5.0"></a>
+## [v0.5.0] - 2022-03-08
+### Fixed
+- Build Fix (#102)
+### Changed
+- setup github actions as ci replacement.
```

Comment:
Just remove this line, it's already captured below as `Migrate to GitHub Actions to gate PRs.`

Reply:
0c23b57e6312e60bef51a98680a6f38b082eae44